experimental/AlgebraicStatistics/src/serialization.jl

```diff
@@ -0,0 +1,58 @@
+@register_serialization_type PhylogeneticModel uses_params 
+@register_serialization_type GroupBasedPhylogeneticModel uses_params 
+
+function save_object(s::SerializerState, pm::PhylogeneticModel)
+    save_data_dict(s) do 
+        save_object(s, pm.graph, :tree) # already implemented 
```

Comment:
does graph(pm) work?  in general you should avoid acessing properties and use functions, and if the functions don't exist you should considering writting them.

Reply:
Now I use functions throughout the code instead of properties.